app/controllers/public/index.js

```diff
@@ -6,10 +6,14 @@ export default Controller.extend({
 
   speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
You can use filterBy to easily filter
Use filterBy: https://www.emberjs.com/api/classes/Ember.computed.html#method_filterBy